client/src/components/formHelpers/form-validators.tsx

```diff
@@ -9,6 +9,14 @@ const fCCRegex =
 const localhostRegex = /localhost:/;
 const httpRegex = /http(?!s|([^s]+?localhost))/;
 
+function hasEndpoint(urlString: string): boolean {
```

Comment:
```suggestion
function isPathRoot(urlString: string): boolean {
```
Maybe? It's not super confusing as is, but it's not technically correct.

You can merge either way, but I thought I'd mention it.